<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>14</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>输出目录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxOutput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutput">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>54</y>
     <width>70</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>xml配置文件</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxOutputXml">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>431</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutputXml">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>浏览...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonStart">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打表</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_outTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>590</width>
     <height>250</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="TableListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>590</width>
     <height>300</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>全部选中</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
